### ISSUE TITLE
is there a way to add the report as a comment to the pr?

### ISSUE THREAD

**Issue by mcquenji** [dorny/test-reporter#313]
I'd like to somehow receive the raw markdown of the report and post it as a comment to the pr. Is there something I missed or is it currently not possible?

**Comment by abenhamdine** [comment]
Looks like it's not currrently possible
There's a (old) fork implementing it : https://github.com/n-ryu/test-reporter

**Comment by owattley-rotageek** [comment]
Yes it would be REALLY useful to just give us a file of the markdown output so we can use as we wish (e.g. for a PR comment in a subsequent step, or saving it somewhere... anything really). The current implementation is great but it seems like an easy and obvious addition to output the actual markdown.

**Comment by jozefizso** [comment]
We will welcome an implementation for this issue.

**Comment by milkyware** [comment]
Just stumbled on this looking for the same. If outputting the summary proposed in #607 is added, I found another [action](https://github.com/marketplace/actions/sticky-pull-request-comment) for adding a sticky comment to a PR

**Comment by davide-ventriglia** [comment]
Hi all! I see the PR is still open
any chance to have this feature in the near future ? I would really benefit from an output that return the entire summary to other steps/jobs

**Comment by R-Raposo** [comment]
It seems [this suggested implementation](https://github.com/dorny/test-reporter/pull/607) doesn't quite work, at least according to the tests I did while calling the action. I've forked the repo and applied some different changes of my own, then packaged it and managed to check that the summary is outputted from the action on my workflows.

Would it be ok for me to create a PR for this?

Edit: I took the liberty to [create a PR](https://github.com/dorny/test-reporter/pull/659)
